src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -250,6 +252,22 @@ public void testPullPrivateRepoWithoutAuth() throws Exception {
   }
 
   @Test
+  public void testBuildImageIdWithBuildargs() throws Exception {
```

Comment:
I'm wondering if this test is testing anything meaningful. A better test might be to use one of the query params like t or cpusetcpus and see they work.